### PR TITLE
PAL-1059 initial move of files from deployment to deployment-k8s

### PR DESCRIPTION
**Pull request by developer01189998819991197253**

### CODE REVIEW

**Review by pd104923 — CHANGES_REQUESTED** [review]
A few comments and questions

File: deployment-k8s/local-k8s/performance/runK8sPerformanceTest.sh [hunk 1]
```diff
@@ -18,8 +18,8 @@ NAMESPACE=$1
 if [ -z "$NAMESPACE" ]
 then
   # If the user doesnt pass in a namespace
-  kubectl exec $(kubectl get pods | awk '/performance/ {print $1}') -- bash -c "cd /usr/share/performance && bash ./runK8sPerformanceTest.sh"
+  kubectl exec "$(kubectl get pods | awk '/performance/ {print $1}')" -- bash -c "cd /usr/share/performance && bash ./runK8sPerformanceTest.sh"
 else
   # If the user passes in a namespace, use the namespace in the kubectl command
-  kubectl exec $(kubectl get pods --namespace="$NAMESPACE" | awk '/performance/ {print $1}') --namespace="$NAMESPACE" -- bash -c "cd /usr/share/performance && bash ./runK8sPerformanceTest.sh"
+  kubectl exec "$(kubectl get pods --namespace="$NAMESPACE" | awk '/performance/ {print $1}')" --namespace="$NAMESPACE" -- bash -c "cd /usr/share/performance && bash ./runK8sPerformanceTest.sh"
```
[the remark this answers is not in the document]
Comment (developer01189998819991197253) [reply deployment-k8s/local-k8s/performance/runK8sPerformanceTest.sh:24]: its genrally a shell reccomendation, prevents word splitting, bit can be taken or left

**Review by pd104923 — CHANGES_REQUESTED** [review]
Just createHRData.sh that is not exectuable now

**Review by pd104923 — APPROVED** [review]

File: deployment-k8s/src/main/resources/k8s-bash-scripts/copyExampleData.sh [hunk 2]
```diff
@@ -45,15 +45,15 @@ rm -rfv "$CLASSPATH"
 echo "Removed previous run classpath-jars for $CLASSPATH"
```
Comment (dev930018) [inline deployment-k8s/src/main/resources/k8s-bash-scripts/copyExampleData.sh:45]: Check the mode on this file, it's been changed from 755 to 644 (no execute bit set)

**Review by dev930018 — CHANGES_REQUESTED** [review]

File: deployment-k8s/Dockerfile [hunk 3]
```diff
@@ -27,14 +27,16 @@ RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/${K8_VERS
...
+COPY target/example /usr/share/deployment-k8s/classpath-jars/example
+COPY target/example /usr/share/deployment-k8s/classpath-jars/performance-test
+COPY target/s3 /usr/share/deployment-k8s/classpath-jars/s3
 
 # Make everything readable
-RUN chmod ugo+r -R /usr/share/deployment
+RUN chmod ugo+r -R /usr/share/deployment-k8s
 
-LABEL service="deployment" project="palisade"
+LABEL service="deployment-k8s" project="palisade"
```
[the remark this answers is not in the document]
Comment (developer01189998819991197253) [reply deployment-k8s/Dockerfile:42]: it was just a consistency thing

**Review by dev930018 — APPROVED** [review]

**Pull request merged**